src/test/unit/components/validator.test.ts

```diff
@@ -317,7 +317,7 @@ describe('Validation', () => {
     ])('Check file format %o', ({ fileName, expected }) => {
       const newFile = mockFile;
       newFile.originalname = fileName;
-      expect(hasValidFileFormat(newFile)).toEqual(expected);
+      expect(hasInvalidFileFormat(newFile)).toEqual(expected);
```

Comment:
Sorry to be annoying but could you please add more tests here. E.g. json file, email file, file name with parenthesis. Can't have too many :)